bundles/com.espressif.idf.core/src/com/espressif/idf/core/util/IDFUtil.java

```diff
@@ -637,4 +658,54 @@ public static IProject getProjectFromActiveLaunchConfig() throws CoreException
 		
 		return null;
 	}
+	
+	public static String getGitExecutablePathFromSystem()
+	{
+		IPath gitPath = new SystemExecutableFinder().find("git"); //$NON-NLS-1$
+		Logger.log("GIT path:" + gitPath); //$NON-NLS-1$
+		if (gitPath != null)
+		{
+			return gitPath.toOSString();
+		}
+		else
+		{
```

Comment:
We can avoid this nested `else` block(not inside code) as you have anyway returning before this at line 668, this will improve readability of code.

Reply:
done